extract_i18n.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+find . -name '*.mo' -delete
+package_name="springboard_iogt"
+mkdir -p ${package_name}/locale
+
+python setup.py extract_messages -o ${package_name}/locale/${package_name}.pot
+
+for locale in "$@"
```

Comment:
`./extract_i18n por_PT eng_GB` if por_PT and eng_GB don't exist yet, otherwise just `./extract_i18n` to update existing message files.

This is a quick adaptation of the script in unicore-cms. I'm going to turn it into a springboard command instead over here: https://github.com/universalcore/springboard/compare/feature/issue-40-i18n-setup-and-commands?expand=1